### ISSUE TITLE
Add labels for medias (accessibility)

### ISSUE THREAD

**Issue by pyg77** [inovector/mixpost#107]
### Steps to reproduce the problem

1. create a post
2. add a media from Media Library

### Expected behaviour

Should be possible to add a label description for each media

### Actual behaviour

No way to add a label

### Detailed description

Should be great for accessibility if we could add labels to medias in Media Library.

### Specifications

Mixpost 2.1.1.0 (selfhosted) Firefox 127.0.1

**Comment by lao9s** [comment]
What is the use case of media labels?

**Comment by pyg77** [comment]
:facepalm: Sorry, I translated too quickly :-/ 
I was talking about the text description of images/medias added to Mixpost, particularly for accessibility reasons.
I haven't found a way of adding such a description in mixpost (and this has been deservedly criticised by our users).

https://www.w3.org/WAI/tutorials/images/

**Comment by lao9s** [comment]
@pyg77 Thank you for the feedback.
